src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -344,11 +346,80 @@ FlutterPluginRegistrant.framework,
 and any plugin frameworks into your existing application
 as described in Option B.
 
+## Local Network Privacy Permissions
+On iOS 14 and higher, enable the Dart multicast DNS
+service in the Debug version of your app
+to allow Flutter debugging functionalities such as hot-reload and DevTools.
```

Comment:
maybe add debugging functionalities such as hot reload and DevTools via `flutter attach`. See more on debugging in https://flutter.dev/docs/development/add-to-app/debugging.

Reply:
Changed it to 
```
to add [debugging functionalities such as hot-reload and DevTools][] via `flutter attach`.
```
with that link.